standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -306,7 +306,7 @@ private void startLocalSegmentStore(int segmentStoreId) throws Exception {
                         .with(ServiceConfig.REST_KEYSTORE_PASSWORD_FILE, this.keyPasswordFile)
                         .with(ServiceConfig.CERT_FILE, this.certFile)
                         .with(ServiceConfig.ENABLE_TLS_RELOAD, this.enableTlsReload)
-                        .with(ServiceConfig.LISTENING_IP_ADDRESS, LOCALHOST)
+                        .with(ServiceConfig.LISTENING_IP_ADDRESS, "0.0.0.0")
```

Comment:
Should the value of the constant be changed?

Reply:
@tkaitchuck I think we need two constants, one for LOCALHOST, another for ALL_INTERFACES. I'm about to push my changes for `ServiceConfig.PUBLISHED_IP_ADDRESS`...